inscriptions/ISic1074.xml

```diff
@@ -178,42 +183,18 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI140554">
-                <ab>
-                    <lb n="1"/>
-
-
-                    Λανθάνουσα ∙ καὶ Ἀγάθη ∙ χρη-<lb n="2"/>στὴ καὶ ἄμεμπτος ∙ ἔζησεν ❦<lb n="3"/>ἔτη #⁵⁶ ο∙β #⁵⁶ μῆνες
-                    δ’ ἔτι καὶ #⁵⁶ γʹ #⁵⁶ ἐπώι#⁵⁶-<lb n="4"/>σε δὲ θυγάτηρ ἰδίᾳ μητρὶ εὐσεβῇ.<lb n="5"/>εἴ τις δὲ ταύτην
-                    ἠδίκησεν καὶ ἐμὲ<lb n="6"/>ἐλύπησεν, μὴ λάθ<supplied reason="omitted">ο</supplied>ιτ<supplied reason="omitted">ο
-                </supplied> τ<supplied reason="omitted">ὸ</supplied>ν θε<supplied reason="omitted">ό</supplied>ν.
-                    <lb n="7"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI175612">
+            <div type="edition" xml:space="preserve" xml:lang="grc" resp="VM">
                 <ab>
-                    <lb n="1"/>
-
-
-                    Λανθάνουσα ∙ <supplied reason="omitted">ἡ</supplied> καὶ Ἀγάθη ∙ χρη-<lb n="2"/>στὴ καὶ ἄμεμπτος ∙
-                    ἔζησεν ❦<lb n="3"/>ἔτη ∙ ο∙βʹ ∙ μῆνες δʹ ἔτι καὶ ∙ γʹ ∙ ἐποί<supplied reason="omitted">η</supplied>∙-
-                    <lb n="4"/>σε δὲ θυγάτηρ ἰδίᾳ μητρὶ εὐσεβή<supplied reason="omitted">ς</supplied>.<lb n="5"/>εἴ τις
-                    δὲ ταύτην ἠδίκησεν καὶ ἐμὲ<lb n="6"/>ἐλύπησεν, μὴ λάθοιτο τὸν θεόν.
-                    <lb n="7"/>
-                </ab>
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI330074">
-                <ab>
-                    <lb n="1"/>
-
-
-                     . . . ἐποί<supplied reason="omitted">η</supplied>-<lb n="2"/>σε
-                    δὲ θυγάτηρ ἰδίᾳ μητρὶ εὐσεβῇ.
-                    <lb n="3"/>
+                    <lb n="1"/>Λανθάνουσα <g type="interpunct">·</g> <supplied reason="omitted">ἡ</supplied>καὶ Ἀγάθη χρη
+                    <lb n="2" break="no"/>στὴ καὶ ἄμεμπτος <g type="interpunct">·</g> ἔζησεν <g ref="http://sicily.classics.ox.ac.uk/alists/listglyph.xml#leaf">❦</g>
+                    <lb n="3"/>ἔτη <hi rend="supraline"><num value="72">οβ</num></hi> μῆνες <hi rend="supraline"><num value="4">δ</num></hi> ἔτι καὶ <hi rend="supraline"><num value="3">γ</num></hi> · ἐποί<supplied reason="omitted">η</supplied>
```

Comment:
supraline is only over the omicron, not the beta as well, in th first numeral; none over the second numeral (delta), or the third (gamma). NB the supraline is recording of a feature on the text, not simply part of the default mark-up of a numeral. The ' included in edited texts is simply indicator of a numeral (equivalent to our `<num>` element), not a reflection of the existence of a supraline or similar.